app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    person.present? && (person.admin? || person.presentations.include?(record))
```

Comment:
So `.present?` can be a little bit omniscient since it also considers empty string and and empty arrays to be falsy. 
In this case. the person will either be a truthy value or nil. do you really need to check for presence? 

Also ruby 2.3 introduced the guard operator which was discussed in stembolt slack at some point last week